hipify-clang/CMakeLists.txt

```diff
@@ -4,87 +4,78 @@ project(hipify-clang)
 set(BUILD_HIPIFY_CLANG 0 PARENT_SCOPE)
 
 # Find LLVM package
-find_package(LLVM 3.8 QUIET PATHS ${HIPIFY_CLANG_LLVM_DIR} NO_DEFAULT_PATH)
-if (NOT ${LLVM_FOUND})
-    find_package(LLVM 3.9 QUIET PATHS ${HIPIFY_CLANG_LLVM_DIR} NO_DEFAULT_PATH)
-    if (NOT ${LLVM_FOUND})
-        message(STATUS "hipify-clang will not be built. To build it please specify absolute path to LLVM 3.8 or LLVM 3.9 package using -DHIPIFY_CLANG_LLVM_DIR")
-    endif()
-endif()
-if (${LLVM_FOUND})
-    list(APPEND CMAKE_MODULE_PATH ${LLVM_CMAKE_DIR})
-    include(AddLLVM)
+find_package(LLVM PATHS ${HIPIFY_CLANG_LLVM_DIR} REQUIRED)
 
-    message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
```

Comment:
Please leave this message if LLVM_FOUND for communicating with users and build engineers.

Reply:
Note that I removed the `QUIET` argument from the call to `find_package(llvm)`. The LLVM finder-system prints:

```
-- Found LLVM 5.0.0
```

All by itself.

Do you want two messages?

Reply:
No, single is enough :+1: